src/s1reader/s1_reader.py

```diff
@@ -590,31 +616,28 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
         osv_list = []
 
     # load individual burst
+    d_seconds = 0.5 * (n_lines - 1) * azimuth_time_interval
```

Comment:
```suggestion
    half_burst_in_seconds = 0.5 * (n_lines - 1) * azimuth_time_interval
```
nit - something more descriptive? It was not immediately obvious what `d_seconds` represented.

Reply:
sure i'll change that, i moved it from the currently named variable https://github.com/opera-adt/s1-reader/blob/main/src/s1reader/s1_reader.py#L612